src/js/popup.js

```diff
@@ -73,41 +88,80 @@ function loadTemplateEditor (callback = false) {
             $('#templateEditorTitle').text('Templates:');
 
             // Sort Alphabetically except with DEFAULT TEMPLATE at the top.
-            templates = sortObject(templates);
+            var templatesArray = sortTemplates(templates);
 
             // Build the Collapsible Template Editor.
-            $.each(templates, function (key, template) {
-                var templateTitle = '';
-                if (key === 'DEFAULT TEMPLATE') {
-                    templateTitle = '<div class="collapsible-header grey lighten-5" data-issueFieldType="' + template['issuetype-field'] + '" style="font-weight: bold;"><i class="material-icons">expand_less</i>' + key + '</div>';
-                } else {
-                    templateTitle = '<div class="collapsible-header grey lighten-5" data-issueFieldType="' + template['issuetype-field'] + '"><i class="material-icons">expand_less</i>' + key + '</div>';
+            $.each(templatesArray, function (index, template) {
+                var boldStyle = '';
+                if (!template['issuetype-field'] && !template['projects-field']) {
+                    boldStyle = 'style="font-weight: bold;"';
                 }
 
+                var templateTitle =
```

Comment:
Ooh that definitely looks a lot nicer! As far as I can tell it is just vanilla JS so I'll make that change.